### ISSUE TITLE
Only use class with composes

### ISSUE THREAD

**Issue by eddiemoore** [atfzl/eslint-plugin-css-modules#13]
If you only use a class that is composing from another class within the same file it will say that the composed class is unused.
```css
.round {
  border-radius: 50%;
}

.myThing {
  composes: round;
  background-color: red;
}
```

In this case if I only use `.myThing` in a component, the plugin will say that `.round` is never used.
Hope that makes sense.

**Comment by atfzl** [comment]
Hi @eddiemoore,

You have you add a trailing `.` before the classname while using `composes`, like this:

```css
.myThing {
  composes: .round;
  background-color: red;
}
```

**Comment by eddiemoore** [comment]
@atfzl Hmmm cause without the `.` is still valid and works. See https://github.com/css-modules/css-modules#composition

**Comment by atfzl** [comment]
Hmm, didn't notice that. I'll update to take classNames without the trailing `.`, in the meantime please use the classnames with `.`

**Comment by eddiemoore** [comment]
Webpack doesn't even want to compile the css with having the `.` at the start of the composes 😢

**Comment by atfzl** [comment]
You are absolutely right.
`composes` does not need the `.`, i got it mixed with `@extend`.

Thanks for opening this issue

Fixed in v2.7.1

**Comment by eddiemoore** [comment]
👍
